### PR TITLE
Feature/update csl

### PR DESCRIPTION
**Pull request by terrasea2**
Was a problem with the way APA citation csl file did things

Trello Card: https://trello.com/c/KENIUrLV/2756-3-1443-update-csl-for-all-jnls-and-update-json-sent-to-the-csl

### CODE REVIEW

File: citeproc/data/styles/bluebook.csl [hunk 1]
```diff
@@ -222,4 +222,3 @@
     </layout>
   </citation>
 </style>
-
```
Comment (simoncrowe) [inline citeproc/data/styles/bluebook.csl]: Are XML files not meant to have a newline at the end?

**Comment by terrasea2** [comment]
Hypno Toad

File: citeproc/data/styles/bluebook.csl (hunk 1 above)
Comment (terrasea2) [reply citeproc/data/styles/bluebook.csl]: Doesn't really matter

Comment (rowan08) [reply citeproc/data/styles/bluebook.csl]: In theory, all files should: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#answer-729795

**Pull request merged**